project/app/src/main/java/com/example/cmput301w21t25/activities_experiments/CreateExperimentActivity.java

```diff
@@ -88,37 +89,47 @@ public void onClick(View v) {
 
```

Comment:
This is a pretty huge method, we should try to refactor this if we have time (not high priority right now though)